### ISSUE TITLE
[webview_flutter] can not scroll and click after three point screenshot

### ISSUE THREAD

**Issue by byh000** [flutter/flutter#90032]
## Steps to Reproduce

1. Run `flutter create bug`.
2. Update the files as follows: ... 
3. ... 
   1> Set WebView.platform = SurfaceAndroidWebView();
   2> do three point screenshot action
   3> scroll or click webview 

**Expected results:** 
I can scroll and click on the content in the `webview`
**Actual results:** 
I cannot scroll or click content in the `webview`

<details>
  <summary>Logs</summary>
 
```console
W/HiTouch_PressGestureDetector(29429): HiTouch Miss: more than two pointers.
E/flutter (29429): [ERROR:flutter/lib/ui/ui_dart_state.cc(186)] Unhandled Exception: Null check operator used on a null value
E/flutter (29429): #0      _AndroidMotionEventConverter.toAndroidMotionEvent (package:flutter/src/services/platform_views.dart:589:31)
E/flutter (29429): #1      AndroidViewController.dispatchPointerEvent (package:flutter/src/services/platform_views.dart:861:31)
E/flutter (29429): #2      _PlatformViewGestureRecognizer.handleEvent (package:flutter/src/rendering/platform_view.dart:559:26)
E/flutter (29429): #3      PointerRouter._dispatch (package:flutter/src/gestures/pointer_router.dart:93:12)
E/flutter (29429): #4      PointerRouter._dispatchEventToRoutes.<anonymous closure> (package:flutter/src/gestures/pointer_router.dart:138:9)
E/flutter (29429): #5      _LinkedHashMapMixin.forEach (dart:collection-patch/compact_hash.dart:397:8)
E/flutter (29429): #6      PointerRouter._dispatchEventToRoutes (package:flutter/src/gestures/pointer_router.dart:136:18)
E/flutter (29429): #7      PointerRouter.route (package:flutter/src/gestures/pointer_router.dart:122:7)
E/flutter (29429): #8      GestureBinding.handleEvent (package:flutter/src/gestures/binding.dart:381:19)
E/flutter (29429): #9      GestureBinding.dispatchEvent (package:flutter/src/gestures/binding.dart:361:22)
E/flutter (29429): #10     RendererBinding.dispatchEvent (package:flutter/src/rendering/binding.dart:278:11)
E/flutter (29429): #11     GestureBinding._handlePointerEventImmediately (package:flutter/src/gestures/binding.dart:316:7)
E/flutter (29429): #12     GestureBinding.handlePointerEvent (package:flutter/src/gestures/binding.dart:280:5)
E/flutter (29429): #13     GestureBinding._flushPointerEventQueue (package:flutter/src/gestures/binding.dart:238:7)
E/flutter (29429): #14     GestureBinding._handlePointerDataPacket (package:flutter/src/gestures/binding.dart:221:7)
E/flutter (29429): #15     _rootRunUnary (dart:async/zone.dart:1370:13)
E/flutter (29429): #16     _CustomZone.runUnary (dart:async/zone.dart:1265:19)
E/flutter (29429): #17     _CustomZone.runUnaryGuarded (dart:async/zone.dart:1170:7)
E/flutter (29429): #18     _invoke1 (dart:ui/hooks.dart:180:10)
E/flutter (29429): #19     PlatformDispatcher._dispatchPointerDataPacket (dart:ui/platform_dispatcher.dart:276:7)
E/flutter (29429): #20     _dispatchPointerDataPacket (dart:ui/hooks.dart:96:31)
E/flutter (29429): 
```

 
  <summary>flutter analyze</summary>

```console
bogon:webview_demo byh000$ flutter analyze
Analyzing webview_demo...                                               
No issues found! (ran in 3.8s)
```

  <summary>flutter doctor -v</summary>

```console
[✓] Flutter (Channel stable, 1.22.6, on macOS 11.4 20F71 darwin-x64, locale zh-Hans-CN)
    • Flutter version 1.22.6 at /Users/byh000/Library/Android/flutter
    • Framework revision 9b2d32b605 (8 months ago), 2021-01-22 14:36:39 -0800
    • Engine revision 2f0af37152
    • Dart version 2.10.5
    • Pub download mirror https://pub.flutter-io.cn
    • Flutter download mirror https://storage.flutter-io.cn

[!] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Users/byh000/Library/Android/sdk
    • Platform android-30, build-tools 30.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)
    ✗ Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/docs/get-started/install/macos#android-setup for more details.

[✗] Xcode - develop for iOS and macOS
    ✗ Xcode installation is incomplete; a full installation is necessary for iOS development.
      Download at: https://developer.apple.com/xcode/download/
      Or install Xcode via the App Store.
      Once installed, run:
        sudo xcode-select --switch /Applications/Xcode.app/Contents/Developer
        sudo xcodebuild -runFirstLaunch
    ✗ CocoaPods not installed.
        CocoaPods is used to retrieve the iOS and macOS platform side's plugin code that responds to your plugin usage on the Dart side.
        Without CocoaPods, plugins will not work on iOS or macOS.
        For more info, see https://flutter.dev/platform-plugins
      To install:
        sudo gem install cocoapods

[✓] Android Studio (version 4.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin installed
    • Dart plugin version 202.8488
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)

[!] IntelliJ IDEA Community Edition (version 2019.2.1)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • For information about installing plugins, see
      https://flutter.dev/intellij-setup/#installing-the-plugins

[✓] Connected device (1 available)
    • YOK AN10 (mobile) • AWMCUT1329003244 • android-arm64 • Android 10 (API 29)

! Doctor found issues in 3 categories.

```
</details>

**Comment by Mayank-9018** [comment]
Could you please upload minimal, reproducible code sample, and maybe a screen record or better description of the issue?

**Comment by danagbemava-nc** [comment]
Hi @byh000, 

Can you please provide your `flutter doctor -v` and `flutter run --verbose` logs, and a [minimal complete reproducible code sample](https://stackoverflow.com/help/minimal-reproducible-example)?

Also, please provide which version of `webview_flutter` you're using. 

One more thing, I noticed you're on an older version of flutter. 

There is the possibility that your issue has been fixed in latest versions of `webview_flutter` but they may not be compatible with the current version of flutter that you're using. 

Kindly upgrade to the latest stable and after fixing all the breaking changes that appear in your code. Please try again. 

Thank you

**Comment by byh000** [comment]
Hi @danagbemava-nc  My webview_flutter version is webview_flutter: ^2.0.13

flutter doctor -v result  can see in  Logs

Code Demo :

import 'dart:io';
import 'package:flutter/material.dart';
import 'package:webview_flutter/webview_flutter.dart';
void main() {
  runApp(MyApp());
}
class MyApp extends StatelessWidget {
  // This widget is the root of your application.
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      theme: ThemeData(
        primarySwatch: Colors.blue,
      ),
      home: MyHomePage(title: 'Flutter Demo Home Page'),
    );
  }
}

class MyHomePage extends StatefulWidget {
  MyHomePage({Key key, this.title}) : super(key: key);

  final String title;
  @override
  _MyHomePageState createState() => _MyHomePageState();
}
class _MyHomePageState extends State<MyHomePage> {
  var _initialUrl = "https://www.baidu.com";
  @override
  void initState() {
    // TODO: implement initState
    super.initState();
    if (Platform.isAndroid) {
      WebView.platform = SurfaceAndroidWebView();
    }
  }
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text(widget.title),
      ),
      body: Center(
        child:WebView(
    initialUrl: _initialUrl,
    javascriptMode: JavascriptMode.unrestricted,))
    );
  }
}

**Comment by byh000** [comment]
issue video 、flutter run --verbose result、demo main.dart  at git@github.com:byh000/test.git

**Comment by danagbemava-nc** [comment]
Hi @byh000, `flutter 2.0` is not the latest version of flutter `stable`. `flutter 2.5` is the latest version of `stable`. Kindly upgrade and see if you still experience this issue. 

Thank you

**Comment by byh000** [comment]
Hi @danagbemava-nc  flutter 2.5  doesn't work

Dart SDK version: 2.14.0 (stable) (Mon Sep 6 13:47:59 2021 +0200) on "macos_x64"
bogon:webview_demo byh000$ flutter doctor -v
[✓] Flutter (Channel stable, 2.5.0, on macOS 11.4 20F71 darwin-x64, locale zh-Hans-CN)
    • Flutter version 2.5.0 at /Users/byh000/dev/flutter_2.5.0
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 4cc385b4b8 (8 days ago), 2021-09-07 23:01:49 -0700
    • Engine revision f0826da7ef
    • Dart version 2.14.0
    • Pub download mirror https://pub.flutter-io.cn
    • Flutter download mirror https://storage.flutter-io.cn

[!] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Users/byh000/Library/Android/sdk
    ✗ cmdline-tools component is missing
      Run `path/to/sdkmanager --install "cmdline-tools;latest"`
      See https://developer.android.com/studio/command-line for more details.
    ✗ Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/docs/get-started/install/macos#android-setup for more details.

[✗] Xcode - develop for iOS and macOS
    ✗ Xcode installation is incomplete; a full installation is necessary for iOS development.
      Download at: https://developer.apple.com/xcode/download/
      Or install Xcode via the App Store.
      Once installed, run:
        sudo xcode-select --switch /Applications/Xcode.app/Contents/Developer
        sudo xcodebuild -runFirstLaunch
    ✗ CocoaPods not installed.
        CocoaPods is used to retrieve the iOS and macOS platform side's plugin code that responds to your plugin usage on the Dart side.
        Without CocoaPods, plugins will not work on iOS or macOS.
        For more info, see https://flutter.dev/platform-plugins
      To install see https://guides.cocoapods.org/using/getting-started.html#installation for instructions.

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)

[✓] IntelliJ IDEA Community Edition (version 2019.2.1)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart

[✓] Connected device (2 available)
    • YOK AN10 (mobile) • AWMCUT1329003244 • android-arm64  • Android 10 (API 29)
    • Chrome (web)      • chrome           • web-javascript • Google Chrome 93.0.4577.82

! Doctor found issues in 2 categories.

**Comment by danagbemava-nc** [comment]
I cannot reproduce this on my device. 

Perhaps it's a device specific issue. 

Labelling for visibility.

<details>
<summary>video</summary>

[IMG_2940.MOV.zip](https://github.com/flutter/flutter/files/7178385/IMG_2940.MOV.zip)

</details

<details>
<summary>flutter doctor -v</summary>

```console
[✓] Flutter (Channel stable, 2.5.0, on macOS 11.5.1 20G80 darwin-arm, locale en-GB)
    • Flutter version 2.5.0 at /Users/nexus/dev/sdks/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 4cc385b4b8 (8 days ago), 2021-09-07 23:01:49 -0700
    • Engine revision f0826da7ef
    • Dart version 2.14.0

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /Users/nexus/Library/Android/sdk
    • Platform android-31, build-tools 31.0.0
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.5.1, Build version 12E507
    • CocoaPods version 1.11.0

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2020.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[✓] VS Code (version 1.60.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.26.0

[✓] Connected device (5 available)
    • Nexus’ IPhone (mobile)       • 00008020-001875E83A38002E            • ios            • iOS 14.7.1 18G82
    • iPhone 12 (mobile)           • F2592D43-5FE3-46BB-8A2B-A38BA468DCE9 • ios            • com.apple.CoreSimulator.SimRuntime.iOS-14-5 (simulator)
    • iPad Pro (9.7-inch) (mobile) • 64BCA5CE-E9C3-47AB-97F3-E0610A3A7E14 • ios            • com.apple.CoreSimulator.SimRuntime.iOS-14-5 (simulator)
    • macOS (desktop)              • macos                                • darwin-arm64   • macOS 11.5.1 20G80 darwin-arm
    • Chrome (web)                 • chrome                               • web-javascript • Google Chrome 93.0.4577.82

• No issues found!
```
</details>

**Comment by byh000** [comment]
Can you tell me your phone device info?

**Comment by danagbemava-nc** [comment]
@byh000 I'm using a `Poco X3 NFC` running `android 11 (MIUI 12.5.2)`

**Comment by byh000** [comment]
I used MI11 Pro android 11, honor V40 , MI 8  all can reprodce this problem

**Comment by byh000** [comment]
@danagbemava-nc  sorry, I made a mistake 。 The issue has been fixed at flutter 2.5.0 。Thank you  very much for your support !

**Comment by danagbemava-nc** [comment]
Closing based on [this comment](https://github.com/flutter/flutter/issues/90032#issuecomment-922258408).
